README.md

```diff
@@ -180,8 +180,11 @@ logic, but without autofix.
 ```go
 ❌   assert.Nil(t, err)
      assert.NotNil(t, err)
-     assert.Equal(t, err, nil)
-     assert.NotEqual(t, err, nil)
+     assert.Equal(t, nil, err)
```

Comment:
I tried to make examples as short as possible, because a lot of combinations here.
Grep `// And other variations` for examples.

If you found such changes useful I can add `Exactly` and `EqualValues` to the other checkers description too

Reply:
Introduced in https://github.com/Antonboom/testifylint/pull/49